src/Steps/Release/PublishRelease.php

```diff
@@ -288,6 +290,132 @@ protected function updateGithubChangelog(OutputInterface $output, LibraryRelease
         $this->log($output, "Github API update failed for <info>" . $library->getName() . "</info>", "error");
     }
 
+    /**
+     * Use the GitHub API to get the release notes for this release
+     */
+    private function getReleaseNotes(OutputInterface $output, GithubClient $client, LibraryRelease $release): string
+    {
+        $this->log($output, 'Getting release notes from GitHub API');
+
+        // Get arguments to send to GitHub API
+        $version = $release->getVersion();
+        $args = [
+            'tag_name' => $version->getValue(),
+            'target_commitish' => "{$version->getMajor()}.{$version->getMinor()}",
+            'previous_tag_name' => $this->getPreviousReleaseTag($output, $release),
+        ];
+
+        // Get release notes from GitHub API
+        $slug = $release->getLibrary()->getGithubSlug();
+        list($org, $repo) = explode('/', $slug);
+        $reponse = $client->getHttpClient()->post(
```

Comment:
What happens when something like a transmission error or not enough github permission occurs?

Reply:
An exception is thrown by the API abstraction